client/extensions/woocommerce/app/settings/payments/payment-method-item.js

```diff
@@ -67,9 +71,43 @@ class PaymentMethodItem extends Component {
 		this.props.changePaymentMethodField( this.props.site.ID, field, value );
 	}
 
-	onSave = () => {
+	onEnableHandler = ( e ) => {
 		const { method, site, translate } = this.props;
+		const enabled = e.target.value === 'yes';
+		this.props.changePaymentMethodEnabled(
+			site.ID,
+			method.id,
+			enabled
+		);
+		const successAction = () => {
+			return successNotice(
+				translate( 'Payment method successfully saved.' ),
+				{ duration: 4000 }
+			);
+		};
+
+		const errorAction = () => {
+			this.props.changePaymentMethodEnabled(
+				site.ID,
+				method.id,
+				! enabled
+			);
+			return errorNotice(
+				translate( 'There was a problem saving the payment method. Please try again.' )
+			);
+		};
 
+		this.props.savePaymentMethodEnabled(
+			site.ID,
+			method,
```

Comment:
If you change this to method.id (which appears to be the only part of method that savePaymentMethodEnabled uses) then the call would be more consistent with changePaymentMethodEnabled and might make maintenance simpler